### PR TITLE
Aggregated the menu and tip totals to RHS

### LINKED ISSUE #51: ability to aggregate group totals

### LINKED ISSUE #74: User persistence
create ui components and connect fe with auth endpoints

- Fix getSessions api
- Login page connect to BE
   - Create and connect apis to get, update menu total and tip total for all users in a session
   - Create and connect apis to get and update menu items for a specific user in a session

### PR DESCRIPTION
**Pull request by christylo**
- When the user clicks "confirm order", the menu total and tip total is updated in database
- Right hand side session menu and tip total values updated from database

TODO: fix bug when a new user tries to join the session, use the mongoose watch function to monitor for changes in the database so all users' group total will be updated once there is a change

### CODE REVIEW

**Comment by acchiang** [comment]
@christylo i'm not sure if these were from your PR but can you also make sure to resolve these linting warnings please? 
image

File: client/src/components/TotalAmount.js [hunk 1]
```diff
@@ -19,11 +20,23 @@ const TotalAmountStyle = styled.div`
...
+      <DollarAmount
+        size={size}
+        label={"Tip Total"}
+        amount={tipAmount}
+      ></DollarAmount>
+      <DollarAmount
+        size={size}
+        label={"Final Total"}
+        amount={(Number(tipAmount) + Number(menuAmount)).toFixed(2)}
+      ></DollarAmount>
```
Comment (acchiang) [inline client/src/components/TotalAmount.js:37]:
```suggestion
      <DollarAmount
        size={size}
        label={"Menu Total"}
        amount={menuAmount}
      />
      <DollarAmount
        size={size}
        label={"Tip Total"}
        amount={tipAmount}
      />
      <DollarAmount
        size={size}
        label={"Final Total"}
        amount={(Number(tipAmount) + Number(menuAmount)).toFixed(2)}
      />
```
nitpick :)

File: server/app.js [hunk 2]
```diff
@@ -31,21 +32,20 @@ require("./config/passport")(passport);
 // Routes
 app.use("/menu", menuRouter);
 app.use("/api/sessions", sessionRouter);
-
+app.use("/user", userRouter);
```
Comment (acchiang) [inline server/app.js]: i think we should follow /api/*s pattern here (i'll update the /menu one), can you update the /user routes? make sure the calls to the user API is also /api/users

**Review by acchiang — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 2 above)
Comment (christylo) [reply server/app.js]: im gonna take out the users stuff we dont actually use them lol

**Pull request merged**